app/src/main/java/me/saket/dank/urlparser/UrlParser.java

```diff
@@ -367,6 +369,22 @@ public static boolean isGooglePlayUrl(String urlHost, String uriPath) {
     return urlHost.endsWith("play.google.com") && uriPath.startsWith("/store");
   }
 
+  public static boolean isYouTubeUrl(String urlHost) {
+       return (urlHost.endsWith("youtube.com") || urlHost.endsWith("youtu.be"));
+  }
+
+  @Nullable
+  public static String getVideoIDFromYouTubeUrl(String url){
```

Comment:
No `get` for getters please. Just `videoIDfromYouTubeUrl()` should be fine.

Reply:
As per the standard style, acronyms (e.g., "ID") are written in camel case as well.

Reply:
Can you add a space before the opening brace (`{`) of this function?